client/lib/faux-plugins.js

```diff
@@ -207,6 +209,15 @@ module.exports = function(roomName) {
       }
     })
 
+    var parseYoutubeTime = function(time){
+       var timeReg = /([0-9]+h)?([0-9]+m)?([0-9]+s?)?/,
+           match   = time.match(timeReg),
+           hours   = parseInt(match[1] || 0),
```

Comment:
For now, i provided the default "0" value outside the function, but i will indeed change that, so that all the checks are done inside the function.